_posts/2020-09-07-witwa-nominations.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
```

Comment:
Is it possible to add a photo in and reference it in the title?

Reply:
I just added it as the image we are using in the post.